.github/workflows/cla.yml

```diff
@@ -0,0 +1,27 @@
+name: "CLA Signature Bot"
```

Comment:
I'd much rather do this in the EIP itself. Maybe something like:

```markdown
## Copyright

Copyright and related rights waived via [CC0](../LICENSE.md). On YYYY/MM/DD, the authors of this EIP agree to the terms of the [Contributor Agreement](../CONTRIBUTING.md).
```

Not sure if that meets the formal requirements for a signature though.

---

Even if we don't embed the "signature" in the EIP itself, we need to get CLAs signed by each author, not just contributors to the repository. Probably would need to modify the bot for that.

Reply:
> Not sure if that meets the formal requirements for a signature though.

No, it wouldn't. I can't just add bill gates to the author list, stick windows 11 in the assets folder, and declare windows 11 to be public domain. The bot would have to be modified.